### ISSUE TITLE
Can't install eza v0.19.1 with mise

### ISSUE THREAD

**Issue by lx4r** [lwiechec/asdf-eza#8]
**Describe the bug**

I'm using this asdf plugin via [mise](https://mise.jdx.dev/) to install and update eza. Until today this has worked without a problem. Now I'm facing issues when trying to install eza v0.19.1.

**Steps to reproduce**

```bash
❯ mise install eza
curl: (22) The requested URL returned error: 404
mise ~/.local/share/mise/plugins/eza/bin/install failed
* Downloading eza release 0.19.1...
curl: (22) The requested URL returned error: 404
asdf-eza: Could not download https://github.com/cargo-bins/cargo-quickinstall/releases/download/eza-0.19.1/eza-0.19.1-aarch64-apple-darwin.tar.gz
asdf-eza: An error ocurred while installing eza 0.19.1.
mise ~/.local/share/mise/plugins/eza/bin/install exited with non-zero status: exit code 1
mise Run with --verbose or MISE_VERBOSE=1 for more information
```

I'm using a Mac with an ARM processor.

**Expected behavior**

Eza is installed and usable.

**Additional context**

As this only started happening with eza v0.19.1 and I see no recent changes to the code of the repo, I suspect that eza might've changed something about their naming scheme.

Thanks in advance for the help :)

**Comment by lwiechec** [comment]
yes, it looks like `cargo-bins` does not have this version of eza published - it looks that v0.19.0 is the last one.

Maybe they are behind with building. I have created a [ticket](https://github.com/cargo-bins/cargo-quickinstall/issues/274). I hope it will be resolved.
